cmd/create/cluster/cmd.go

```diff
@@ -1424,6 +1465,18 @@ func run(cmd *cobra.Command, _ []string) {
 		)
 	}
 
+	if isSTS {
+		reporter.Infof("Operator roles options :")
+		operatorroles.Cmd.Run(operatorroles.Cmd, []string{clusterName, mode})
+		if mode == "manual" {
+			reporter.Infof("Run the following command to create the oidc provider once operator roles are "+
+				"created : \n'rosa create oidc-provider --cluster %s' \n", clusterName)
```

Comment:
You can run both commands - as long as they are in order, we should be fine. That means this entire block can be:
```
if isSTS && mode != "" {
	operatorroles.Cmd.Run(operatorroles.Cmd, []string{clusterName, mode})
	oidcprovider.Cmd.Run(oidcprovider.Cmd, []string{clusterName, mode})
}
```

Reply:
the issue here is if mode is manual the oidc-provider command will error out due to the roles not existing.

Reply:
Then we need to update the oidc-provider command to support being called programmatically. Since manual doesn't write anything to the customer's AWS account, we should be able to skip those checks and just print out the necessary commands.

On the other hand, the fact that we create the roles _first_ and then the OIDC provider is wrong, as when roles are created we add a non-existent OIDC provider in the trust policy. The _real_ order of things _should_ be:
1. create cluster spec
2. create OIDC provider (based on spec data)
3. create operator roles (based on OIDC provider data)
4. add operator roles to cluster spec

This is some of the work @pvasant is doing currenty, so maybe we should align this command to that as well.